### DISCUSSION TITLE
How to increase inertia？

### DISCUSSION THREAD

**Discussion by qq908323236** [WorldWindEarth/WorldWindKotlin#26] [Q&A]
Hi,

How to realize that when I am rolling the earth, I suddenly lift my finger, and the earth will roll for a while, as if there is inertia。
There are pinch, rotation can also be achieved。
Overall, it feels like Google Earth to the touch。

Regards

**Comment by EMaksymenko** [comment]
Currently this functionality is not available.
It requires to be added in BasicWorldwindowController.
You can extend BasicWorldwindowController in your application or create a pull request with implementation.

Basically it needs some coroutine which will continue moving camera after mouse up.

**Reply by qq908323236** [reply]
thank you for your reply.
I get it already, but I still hope you can implement this feature in the future.

**Reply by qq908323236** [reply]
I tried to implement it, but I always had problems because I was not familiar with the project. If you have time later, can you implement this function?

grateful

**Reply by EMaksymenko** [reply]
We have put it in backlog, but this task is not a priority now.

**Reply by qq908323236** [reply]
OK, looking forward to your update.
